addons/l10n_ir_accounting/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# Copyright (C) 2022 Fadoo  (<http://www.Fadoo.ir>).
+
+{
+    "name": "Iran - Accounting",
+    "version": "14.0.3.0.0",
+    "category": "Accounting/Localizations/Account Charts",
+    "summary": """
+Iran Accounting Module
+============================
+iran accounting chart and localization.
+
+Also:
+    - activates a number of regional currencies.
+    - sets up Iran taxes.
+    """,
+    "author": "Fadoo, Odoo Community Association (OCA)",
+    "maintainer": ["saeed-raesi"],
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/l10n-iran",
+    "depends": ["account"],
```

Comment:
you could add the base_vat dependency so that when adding a customer the validity of the VAT is checked

Reply:
Also missing 'l10n_multilang', check in other loca how to do it, it will allow all the COA to be translatable